go_api/dbinit.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"github.com/NUTFes/lottery/go_api/domain"
+
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+func dbinit() {
+	db, err = gorm.Open(mysql.Open(dsn), &gorm.Config{})
+	if err != nil {
+		return
+	}
+	db.Migrator().CreateTable(domain.User{})
```

Comment:
```suggestion
	db.AutoMigrate(domain.User{})
```
下の4つのやつも同じですが、`db.AutoMigrate(domain.User{})`このようにしないと中間テーブルとかは自動生成されないかと思います